src/Sulu/Bundle/PreviewBundle/Preview/Renderer/PreviewKernel.php

```diff
@@ -42,13 +45,33 @@ public function registerContainerConfiguration(LoaderInterface $loader)
     public function getRootDir()
     {
         if (null === $this->rootDir) {
-            $reflectionClass = new \ReflectionClass(\WebsiteKernel::class);
+            $reflectionClass = new \ReflectionClass(WebsiteKernel::class);
             $this->rootDir = dirname($reflectionClass->getFileName());
         }
 
         return $this->rootDir;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getProjectDir()
```

Comment:
moved to https://github.com/sulu/sulu/pull/3980 as this is also a problem for 1.5 and 1.6 that the previewkernel uses false project_dir